extensions/blocks/calendly/calendly.php

```diff
@@ -119,35 +146,52 @@ function load_assets( $attr, $content ) {
 	);
 
 	if ( 'link' === $style ) {
-		wp_enqueue_style( 'jetpack-calendly-external-css', 'https://assets.calendly.com/assets/external/widget.css', null, JETPACK__VERSION );
-
-		/*
-		 * If we have some additional styles from the editor
-		 * (a custom text color, custom bg color, or both )
-		 * Let's add that CSS inline.
-		 */
-		if ( ! empty( $submit_button_text_color ) || ! empty( $submit_button_background_color ) ) {
-			$inline_styles = sprintf(
-				'#%1$s .wp-block-button__link{%2$s%3$s}',
+		if ( ! wp_style_is( 'jetpack-calendly-external-css' ) ) {
+			wp_enqueue_style( 'jetpack-calendly-external-css', 'https://assets.calendly.com/assets/external/widget.css', null, JETPACK__VERSION );
+		}
+		if ( strstr( $content, sprintf( '>%s</a>', $orig_url ) ) ) {
+			// This is the lecacy version, so create the full link content.
```

Comment:
This change works; the block doesn't break when I switch to this branch anymore. I wonder if we should move this to a new function so that its more clearly a deprecation, and easier to remove in the future. That shouldn't block us merging this PR though. I'm happy to make that change if you'd like?